### PR TITLE
Cloud Commander version 1.0.131223

### PR DESCRIPTION
**Pull request by VincentMejan**
- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,6000 @@
...
+                },
+                "resourceProvisioningOptions": {
+                  "type": "array",
+                  "items": {
+                    "description": "Resource provisioning options of the group.",
+                    "title": "Resource provisioning options",
+                    "type": "string"
+                  },
+                  "description": "Resource provsioning options."
+                },
```
Comment (vmanoharas) [inline certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json:1494]: Can you please correct the spelling "provsioning"

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,6000 @@
...
+                  "enum": [
+                    true,
+                    false
+                  ]
+                },
+                "visibility": {
+                  "type": "string",
+                  "description": "Visibility of the group.",
+                  "title": "Visbility"
+                },
```
Comment (vmanoharas) [inline certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json:1508]: Can you please correct the spelling "Visbility"

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,6000 @@
...
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Indicates the password validity in days.",
+                        "title": "Password validity"
+                      },
+                      "passwordNotificationWindowInDays": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Indicates the password notifcation window in days.",
+                        "title": "Password notification windows"
```
Comment (vmanoharas) [inline certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json:1689]: Can you please correct spelling "notifcation"

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,6000 @@
...
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Subscription identifier of the customer.",
+            "x-ms-summary": "Subsription id.",
+            "x-ms-url-encoding": "single"
```
Comment (vmanoharas) [inline certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json:2244]: Can you please correct the spelling "Subsription"

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,6000 @@
...
+          },
+          "500": {
+            "description": "Unexpected error."
+          }
+        }
+      }
+    },
+    "/command/v1/commandblocks/{commandBlockId}": {
+      "get": {
+        "description": "Get the details of a Command Block. Provider and action validation depend on the the specific Command Block.",
```
Comment (vmanoharas) [inline certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json]: can you please remove one "the"

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,6000 @@
...
+                                "title": "Command Set",
+                                "enum": [
+                                  true,
+                                  false
+                                ]
+                              },
+                              "commandBlock": {
+                                "type": "boolean",
+                                "description": "Commmand Block.",
+                                "title": "Command Block",
```
Comment (vmanoharas) [inline certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json:3629]: Can you please correct the spelling "Commmand"

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,6000 @@
...
+          },
+          "500": {
+            "description": "Unexpected error."
+          }
+        }
+      }
+    },
+    "/command/v1/organizations/{organizationId}/commandblocks/{commandBlockId}/jobs/{jobId}": {
+      "get": {
+        "description": "Get details of a single user job, provider and action validation depend on the the specific command block.",
```
Comment (vmanoharas) [inline certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json]: Can you please remove one "the"

File: certified-connectors/N-able Cloud Commander/readme.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+* `Add a member to a Microsoft Entra ID group`: Add a member to a Microsoft 365 group or a security group through the members navigation property
+* `Remove member from a Microsoft Entra ID group`: Remove a member from a group via the members navigation property.
+* `Get Partner Center subscriptions`: Retrieves all subscriptions for the customer of a partner by their active directory tenant identifier.
+* `Update Partner Center subscription quantity`: Update the quantity of a subscription
+* `List Command Blocks`: Retrieve a list of available Command Blocks
+* `Get commandblock jobs for all authorized organizations`: Get commandblock jobs for Command Blocks the calling user is authorized to view.
+* `Get statistics on the Command Block jobs for all authorized organizations`: Get statistics on the Command Block jobs for Command Block the calling user is authorized to view.
+* `Get commandblock jobs of a specific organization`: Get commandblock jobs for Command Blocks the calling user is authorized to view.
+* `Get Command Block details`: Get the details of Command Block, provider and action validation depend on the the specific Command Block.
```
Comment (vmanoharas) [inline certified-connectors/N-able Cloud Commander/readme.md]: can you please remove one "the"

File: certified-connectors/N-able Cloud Commander/readme.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+* `Add a Command Block job for a user`: Add a Command Block job, provider and action validation depend on the specific Command Block.
+* `Add a Command Block job for a device`: Add a Command Block job, provider and action validation depend on the specific Command Block.
+* `Add a Command Block job for a group`: Add a Command Block job, provider and action validation depend on the specific Command Block.
+* `Add a Command Block job for an organization`: Add a Command Block job, provider and action validation depend on the specific Command Block.
+* `Get details of a single user job`: Get details of a single user job, provider and action validation depend on the the specific Command Block.
```
Comment (vmanoharas) [inline certified-connectors/N-able Cloud Commander/readme.md]: can you please remove one "the"

File: certified-connectors/N-able Cloud Commander/apiProperties.json [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+            },
+            "refreshUrl": {
+              "value": "https://login.microsoftonline.com/common/oauth2/v2.0/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#000000",
+    "capabilities": [],
```
Comment (vmanoharas) [inline certified-connectors/N-able Cloud Commander/apiProperties.json:33]: Can you please confirm that the iconBrandColor is correct? 

Please refer this link https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector

File: certified-connectors/N-able Cloud Commander/readme.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
+## Cloud Commander
+Cloud Commander is a unified Microsoft CSP portal that lets you automate common tasks across all Microsoft 365 apps without switching consoles or customers.
```
Comment (vmanoharas) [inline certified-connectors/N-able Cloud Commander/readme.md:2]: Please add all required fields for readme.md file from the below template.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

**Review by vmanoharas — COMMENTED** [review]
Hello @VincentMejan, 

Can you please fix review comments?

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json (hunk 1 above)
Comment (VincentMejan) [reply certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json:1494]: Fixed

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json (hunk 3 above)
Comment (VincentMejan) [reply certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json:1689]: Fixed

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json (hunk 2 above)
Comment (VincentMejan) [reply certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json:1508]: Fixed

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json (hunk 4 above)
Comment (VincentMejan) [reply certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json:2244]: Fixed

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json (hunk 5 above)
Comment (VincentMejan) [reply certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json]: Fixed

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json (hunk 6 above)
Comment (VincentMejan) [reply certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json:3629]: Fixed

File: certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json (hunk 7 above)
Comment (VincentMejan) [reply certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json]: Fixed

File: certified-connectors/N-able Cloud Commander/readme.md (hunk 8 above)
Comment (VincentMejan) [reply certified-connectors/N-able Cloud Commander/readme.md]: Fixed

File: certified-connectors/N-able Cloud Commander/readme.md (hunk 9 above)
Comment (VincentMejan) [reply certified-connectors/N-able Cloud Commander/readme.md]: Fixed

File: certified-connectors/N-able Cloud Commander/apiProperties.json (hunk 10 above)
Comment (VincentMejan) [reply certified-connectors/N-able Cloud Commander/apiProperties.json:33]: Fixed

File: certified-connectors/N-able Cloud Commander/readme.md (hunk 11 above)
Comment (VincentMejan) [reply certified-connectors/N-able Cloud Commander/readme.md:2]: Fixed

**Review by vmanoharas — APPROVED** [review]
Hello @VincentMejan 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Comment by vmanoharas** [comment]
Hello @VincentMejan, 

There is a message for you in ISV studio, can you please address?

**Pull request merged**